overlord/snapshotstate/backend/backend.go

```diff
@@ -179,6 +172,7 @@ func Filename(snapshot *client.Snapshot) string {
 }
 
 func isSnapshotFilename(fname string) (ok bool, setID uint64) {
```

Comment:
(nitpick) I would probably rename the function input far from "fname" to something like "path" or "filePath" or "fullPath" or something. Unfortunately we don't really have a convention for this, i.e. when fname or fn means just the filename or the filename with path.

Reply:
Done.